client/app/components/project-chat/project-chat.directive.js

```diff
@@ -42,11 +42,14 @@
         vm.message = '';
         vm.messages = [];
         vm.maxlengthComment = 250;
+
+        vm.hasNotScrolledYet = true;
```

Comment:
Perhaps consider switching logic to hasScrolled rather than hasNotScrolled to avoid double negative.